### PR TITLE
fix convert support chinese

### PR DESCRIPTION
**Pull request by UDLD**

### CODE REVIEW

File: protobuf/src/main/java/com/citahub/cita/protobuf/ConvertStrByte.java [hunk 1]
```diff
@@ -69,33 +66,41 @@ public static String bytesToHexString(byte[] b) {
...
-        for (int i = 0; i < strPart.length(); i++) {
-            int ch = (int) strPart.charAt(i);
-            String strHex = Integer.toHexString(ch);
-            hexString.append(strHex);
+        char[] chars = "0123456789abcdef".toCharArray();
+        StringBuilder sb = new StringBuilder();
+        byte[] bs = strPart.getBytes();
+        int bit;
+        for (int i = 0;
+             i < bs.length; i++) {
```
Comment (rainchen) [inline protobuf/src/main/java/com/citahub/cita/protobuf/ConvertStrByte.java]: why break into 2 lines?

File: protobuf/src/main/java/com/citahub/cita/protobuf/ConvertStrByte.java [hunk 2]
```diff
@@ -69,33 +66,41 @@ public static String bytesToHexString(byte[] b) {
 
 
     public static String stringToHexString(String strPart) {
-        StringBuffer hexString = new StringBuffer();
```
Comment (rainchen) [inline protobuf/src/main/java/com/citahub/cita/protobuf/ConvertStrByte.java:72]: do we have a test case for this change?

**Review by rainchen — CHANGES_REQUESTED** [review]
commented

File: protobuf/src/main/java/com/citahub/cita/protobuf/ConvertStrByte.java (hunk 2 above)
Comment (UDLD) [reply protobuf/src/main/java/com/citahub/cita/protobuf/ConvertStrByte.java:72]: @rainchen  yes, more test in old cases

File: protobuf/src/main/java/com/citahub/cita/protobuf/ConvertStrByte.java (hunk 1 above)
Comment (UDLD) [reply protobuf/src/main/java/com/citahub/cita/protobuf/ConvertStrByte.java]: fixed

**Comment by UDLD** [comment]
@rainchen  add chinese case done

File: protobuf/src/main/java/com/citahub/cita/protobuf/ConvertStrByte.java (hunk 2 above)
Comment (UDLD) [reply protobuf/src/main/java/com/citahub/cita/protobuf/ConvertStrByte.java:72]: > do we have a test case for this change?

fixed

**Review by rainchen — APPROVED** [review]

**Review by yangzhenlong — APPROVED** [review]
the title `china` should be `chinese`

**Pull request merged**